java ai monitoring pr (#17559)
newrelic/docs-website: src/content/docs/apm/agents/java-agent/api-guides/ai-monitoring.mdx

When you've instrumented your app with AI monitoring, you can access some APIs for collecting token count and user feedback. To use the AI monitoring APIs, check that your Java agent is updated to version 8.12.0 or higher.

> This doc gives you high level procedures for updating your code to access the [token count and user feedback APIs](https://newrelic.github.io/java-agent-api/javadoc/com/newrelic/api/agent/AiMonitoring.html#recordLlmFeedbackEvent). 

Reviewer: Not sure about the use of "high level" here

> This doc gives you high level procedures for updating your code to access the [token count and user feedback APIs](https://newrelic.github.io/java-agent-api/javadoc/com/newrelic/api/agent/AiMonitoring.html#recordLlmFeedbackEvent). 

Author: ```suggestion
This doc provides procedures for updating your code to access the [token count and user feedback APIs](https://newrelic.github.io/java-agent-api/javadoc/com/newrelic/api/agent/AiMonitoring.html#recordLlmFeedbackEvent). 
```

        NewRelic.getAgent().getAiMonitoring.setLlmTokenCountCallback(myTokenCountCallback);
    ```

> To use the callback, implement `LlmTokenCountCallback` so that it returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. If the values are less than or equal to `0`, `LlmTokenCountCallbacks` will not be attached to an event. Keep in mind that this API should be called only once. Calling this API multiple times will replace each previous callback. 

Reviewer: will not > won't

Reviewer: Keep in mind that you should only call this API once

> To use the callback, implement `LlmTokenCountCallback` so that it returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. If the values are less than or equal to `0`, `LlmTokenCountCallbacks` will not be attached to an event. Keep in mind that this API should be called only once. Calling this API multiple times will replace each previous callback. 

Author: ```suggestion
To use the callback, implement `LlmTokenCountCallback` so that it returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. If the values are less than or equal to `0`, `LlmTokenCountCallbacks` won't attach to an event. Keep in mind that you should only call this API once. Calling this API multiple times will replace each previous callback. 
```
